.github/pull_request_template.md

```diff
@@ -1,15 +1,52 @@
-Describe your changes here.
+_Authors: Please fill out this form carefully and completely._
 
-Closes # (type the issue number after # if applicable; otherwise remove this line)
+_Reviewers: By approving this Pull Request you are approving the code change, as well as its deployment and mitigation plans._
+_Please read this description carefully. If you feel there is anything unclear or missing, please ask for updates._
+
+#### List the issues that this change affects.
+<!--Every code change must address _at least 1_ issue. Fixes a bug, completes a task, every change
+      should have a corresponding issue listed here. If one does not already exist, create one. -->
+      
+Closes # (type the GitHub issue number after #)
+
+### What are you trying to accomplish?
+<!-- Provide a description of the changes. -->
 
 ### Screenshots
+<!-- Provide before/after screenshots, videos, or graphs for any visual changes; otherwise, remove this section -->
+
+#### Risk Assessment
+  <!-- Please select from one of the following and detail why this level was chosen -->
+
+- [ ] **Low risk** the change is small, highly observable, and easily rolled back.
+- [ ] **Medium risk** changes that are isolated, reduced in scope or could impact few users. The change will not impact library availability.
+- [ ] **High risk** changes are those that could impact customers and SLOs, low or no test coverage, low observability, or slow to rollback.
```

Comment:
I think this section is challenging from the library side as it's hard or limiting to express changes in terms of:

- Observability
- Availability
- The ability of a change to be rolled back
- The impact of a change to SLOs

Instead of evaluating the risk of a change as it relates to a production environment, we could talk about the types of changes that the PR introduces from the perspective of the end user.

```suggestion
## Changelog

<!-- List out API changes that this Pull Request introduces -->

### New

### Changed

### Removed
```

Reply:
Adopted the recommended change and added a section for "rollout strategy": https://github.com/primer/react/pull/3344/commits/62bedc473eff54e11c46cd0a4794e0d9dcb7d310